packages/sdk/src/Confidence.ts

```diff
@@ -314,6 +317,14 @@ export class Confidence implements EventSender, Trackable, FlagResolver {
     return (await this.evaluateFlag(path, defaultValue)).value;
   }
 
+  private showLoggerLink(flag: string, context: Context) {
+    this.config.logger.info?.(
```

Comment:
This is a too high level IMO. 
Would suggest `debug` or maybe even `trace`

Reply:
Chatted with Andreas about it: lower in Confidence class we only put this into info on "dev" builds. If we change it to debug/trace, the current factory instance won't show any of the logs in an app. So the user should then know to create a Confidence instance with a logger in debug mode.

Reply:
I understand, sounds good!